carbon-design-system/carbon
# chore(Tile): updated props for v11

REF #9535

#### Changelog

- Added deprecation warnings for handleClick and handleKeyDown to `Tile`.
- Updated the PublicAPI test snapshot.
- Touched up `Tile` by changing `{...other}` -> `{...rest}` and `classNames` -> `cx`.
- Updated the `Tile` story with new props.
- Added comments in the appropriate areas to remove deprecations for the next major release.

#### Testing / Reviewing

- Check the Tile stories and make sure everything is firing off as it should.
- Make sure tests are passing.
